types/simple-oauth2/index.d.ts

```diff
@@ -1,9 +1,10 @@
-// Type definitions for simple-oauth2 1.1
+// Type definitions for simple-oauth2 1.6
```

Comment:
Please swap this so `2.2` appears at the index, and what's here now is in a `v1` directory.